eng/scripts/MavenPackaging.ps1

```diff
@@ -75,6 +75,7 @@ function Get-AssociatedArtifacts([MavenPackageDetail]$PackageDetail) {
 function Get-SonaTypeProfileID([string]$GroupID) {
   $sonaTypeProfileID = switch -wildcard ($GroupID)
   {
+    "io.clientcore*"                 { "88192f04117501" }
```

Comment:
The ids for io.clientcore and com.azure are the same. They should have different sonatype profile ids.

Reply:
@weshaggard confirmed that this code is not used anymore since we started publishing via ESRP. We could use a blank value too instead.